specification/validated-relationships-service-api.yaml

```diff
@@ -317,57 +322,221 @@ components:
+              description: An identifier - identifies some entity uniquely and unambiguously. Typically this is used for business identifiers.
+              properties:
+                system:
+                  type: string
+                  description: Codesystem URL for the subject's NHS number.
+                  enum: ["https://fhir.nhs.uk/Id/nhs-number"]
+                value:
+                  type: string
+                  description: The subject's NHS number.
+                  example: "9000000002"
         questionnaire:
           type: string
           description: The Questionnaire that defines and organizes the questions for which answers are being provided.
+          example: "Questionnaire/01dc6813-3421-4d14-948d-a4888241add1"
         item:
           type: array
           description: A group or question item from the original questionnaire for which answers are provided.
           items:
             oneOf:
-              - $ref: '#/components/schemas/QuestionnaireItem_ProxyDetails'
-              - $ref: '#/components/schemas/QuestionnaireItem_PatientDetails'
-              - $ref: '#/components/schemas/QuestionnaireItem_RequestedServices'
+              - $ref: '#/components/schemas/QuestionnaireResponseItem_RelatedPerson'
+              - $ref: '#/components/schemas/QuestionnaireResponseItem_ParentalApplicationSupplementaryDetails'
+              - $ref: '#/components/schemas/QuestionnaireResponseItem_Patient'
+              - $ref: '#/components/schemas/QuestionnaireResponseItem_RequestedAccess'
 
-    QuestionnaireItem_ProxyDetails:
+    QuestionnaireResponseItem_RelatedPerson:
       type: object
+      required:
+        - linkId
+        - text
+        - item
       properties:
         linkId:
           type: string
-          enum: ["proxy_details"]
+          enum: ["relatedPerson"]
         text:
           type: string
+          example: "relatedPerson"
         item:
           type: array
           items:
             oneOf:
               - type: object
+                required: 
+                  - linkId
+                  - text
+                  - answer
                 properties:
                   linkId:
                     type: string
-                    enum: ["nhs_number"]
+                    enum: ["relatedPerson_identifier"]
                   text:
                     type: string
                     example: "NHS Number"
                   answer:
                     type: array
+                    minItems: 1
+                    maxItems: 1
                     items:
                       oneOf:
                         - type: object
                           properties:
                             valueString:
                               type: string
-                              example: "9000000005"
+                              example: "9000000001"
+
               - type: object
+                required: 
+                  - relatedPerson_name
```

Comment:
`relatedPerson_name` will only be mandatory if a patient is nominating. Likely will have to check this through logic

Reply:
Removed the required links for name and DOB due to above logic depending on who is the applicant